src/main/java/org/java_websocket/WebSocketImpl.java

```diff
@@ -451,6 +455,9 @@ private void closeConnectionDueToInternalServerError(RuntimeException exception)
   private ByteBuffer generateHttpResponseDueToError(int errorCode) {
     String errorCodeDescription;
     switch (errorCode) {
+      case 401:
+        errorCodeDescription = "401 WebSocket Authentication Failure";
```

Comment:
I don't know if that is binding, but in OCPP 2.0.1, it says "HTTP/401 Authorization Required", so maybe the text should be "401 Authorization Required"?

Reply:
I think in the test it only checks error code but we can change it to be aligned with OCPP 2.0.1

Reply:
done